Wrappers/Python/ccpi/web_viewer/trame_viewer.py

```diff
@@ -0,0 +1,772 @@
+#
+#   Copyright 2022 STFC, United Kingdom Research and Innovation
+#
+#   Author 2022 Samuel Jones
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+
+import os
+
+import matplotlib.pyplot as plt
+from trame import update_layout
+from trame.html import vtk, vuetify
+from trame.layouts import SinglePageWithDrawer
+from vtkmodules.util import colors
+from vtkmodules.vtkIOImage import vtkMetaImageReader
+
+from ccpi.viewer.CILViewer import CILViewer
+from ccpi.viewer.CILViewer2D import SLICE_ORIENTATION_XY, SLICE_ORIENTATION_XZ, SLICE_ORIENTATION_YZ
+from ccpi.viewer.utils.conversion import cilHDF5ResampleReader
+from ccpi.web_viewer.camera_data import CameraData
+
+DEFAULT_SLICE = 32
+INITIAL_IMAGE = "head.mha"
```

Comment:
The head.mha and head.npy files are on this branch - they should be removed

Reply:
They are included in the history, when we squash the commits pre-merge to master it will remove them in the history. The INITIAL_IMAGE is a default that it can actively handle if not present.

Reply:
are you sure they aren't still on the branch because it's werid they appear in the files list and aren't empty files or anything

Reply:
https://github.com/vais-ral/CILViewer/pull/248/files#diff-f08ba911588e0008d1a15a8d793425829e64692c846cb503afc497acdd349cf8

Reply:
Fixed